### PR TITLE
feat: improve minify config

### PR DESCRIPTION
**Pull request by thewh1teagle**
Improve minify config
Not sure if there's a better way to parse it into structured config instead of using `slices.contains` :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,7 @@ ssl:
...
   # Optional
-  minify:
-    js: true
-    html: true
-    css: true
-    json: true
-    xml: true
-    svg: true
+  minify: [js, html, css, json, xml, svg]
     # You can use 'all: true' instaed to enable all content-types
```
Comment (hvuhsg) [inline README.md]: Update comment

File: README.md [hunk 2]
```diff
@@ -141,13 +135,7 @@ services:
...
         
-        minify:
-          js: true
-          html: true
-          css: true
-          json: true
-          xml: true
-          svg: true
+        minify: [js, html, css, json, xml, svg]
           # You can use 'all: true' instaed to enable all content-types
```
Comment (hvuhsg) [inline README.md]: Update comment

**Review by hvuhsg — CHANGES_REQUESTED** [review]
Looks good!, the only issue is the comments

File: README.md (hunk 1 above)
Comment (thewh1teagle) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (thewh1teagle) [reply README.md]: Fixed

**Review by hvuhsg — APPROVED** [review]
Thanks

**Pull request merged**